docs/howto/auth/awscognito.rst

```diff
@@ -71,8 +71,8 @@ able to configure the instance automatically, replace relevant config variables:
 Manual configuration to use the AWS Cognito Oauthenticator
 ============================================================
 
-Assuming tljh has already been installed, we need to make sure the oautheneticator module is at 0.10.0 and if not 
-do a pip install oauthenticator>=0.10.0
+Assuming tljh has already been installed, we need to make sure the oauthenticator module is at 0.10.0 and if not 
+do a pip install oauthenticator==0.10.0
```

Comment:
I think we should also mention that an AWSCongnito authenticator can still be achieved with oauthenticator 0.11, by configuring the GenericOAuthenticator class.

I tried documenting such a configuration in this PR https://github.com/jupyterhub/oauthenticator/pull/336, but I didn't test it however, so it might be incomplete. If it gets accepted, we could later link to the docs.

Reply:
Waiting for https://github.com/jupyterhub/oauthenticator/pull/336 to be merged sounds good :+1:

Just opened https://github.com/jupyterhub/the-littlest-jupyterhub/issues/540 to track this.